test/tools/d_do_test.d

```diff
@@ -755,6 +756,9 @@ int tryMain(string[] args)
             compile_output = compile_output.unifyNewLine();
             compile_output = std.regex.replace(compile_output, regex(`^DMD v2\.[0-9]+.*\n? DEBUG$`, "m"), "");
             compile_output = std.string.strip(compile_output);
+            // replace test_result path with fixed ones
+            compile_output = compile_output.replace(envData.results_dir, "{{RESULTS_DIR}}");
+            compile_output = compile_output.replace(absoluteResultDirPath, "{{RESULTS_DIR}}");
```

Comment:
The makefile uses a relative path (though it's different on the auto-tester) whereas `run.d` uses an absolute one.
This should unify all cases though.